source/index.html.md

```diff
@@ -110,9 +110,11 @@ App Type | ```Provider``` Provider facing app
 FHIR Spec | ```dstu2``` The latest spec version supported by Cerner.
 Authorized | ```Yes``` Authorized App will go through secured OAuth 2 login.
 Standard Scopes | These scopes are required to launch the SMART app.
-User Scopes | None
+User Scopes | None 
 Patient Scopes | Locate the ***Patient Scopes*** table and select the ***Patient*** read and ***Observation*** read scopes.
 
+Specifying user scopes or patient scopes will result a slightly different testing workflow. See this section: [Test your App](#test-your-app).
```

Comment:
> result a slightly 

result in a slightly

We could add link to this [spec](http://hl7.org/fhir/smart-app-launch/scopes-and-launch-context/index.html#scopes-for-requesting-clinical-data) for the differences between User Level Scopes vs. Patient Level Scopes.

Reply:
OK.